xml/gl.xml

```diff
@@ -4948,7 +4948,7 @@ typedef unsigned int GLhandleARB;
         <enum value="0x8D40" name="GL_FRAMEBUFFER" group="ObjectIdentifier,FramebufferTarget,CheckFramebufferStatusTarget"/>
         <enum value="0x8D40" name="GL_FRAMEBUFFER_EXT"/>
         <enum value="0x8D40" name="GL_FRAMEBUFFER_OES" group="FramebufferTarget"/>
-        <enum value="0x8D41" name="GL_RENDERBUFFER" group="ObjectIdentifier,RenderbufferTarget,CopyImageSubDataTarget"/>
+        <enum value="0x8D41" name="GL_RENDERBUFFER" group="ObjectIdentifier,RenderbufferTarget,CopyImageSubDataTarget,TextureTarget"/>
```

Comment:
I'm not sure on this change, as currently `TextureTarget` kind of unofficially means "enums that you can pass to `glTexImage*`" and `GL_RENDERBUFFER` isn't really in that group.

Generally I think we might need to create more groups than just `TextureTarget` as it's also used for `glBindTexture` which doesn't allow you to pass a proxy texture as a target.
Other examples of functions that use `TextureTarget` but specifies a different valid enum set include `glGetInternalformat` (as pointed out in this PR) and `glGetTexLevelParameter`.

Reply:
I'm fine with leaving it like this for this PR (I think), but I think we need to start a discussion on whether we should create more groups for this.

Reply:
Agreed. Creating more groups seems to be the best long-term solution. However, glGetInternalformat, glGetTexLevelParameter, and glBindTexture seem all to use different sets of valid values for the target parameters. This means potentially three new groups. In addition, the valid target values of the different functions depend on the API version.

Instead of doing this complex refactoring, I went for the simpler route -- keep the group that glGetInternalformat is already using (TextureTarget) and add GL_RENDERBUFFER to it.

Reply:
I think the "depending on opengl version" enums is something that should be figured out from the `<feature>` tags.

Having the enums groups be the set of all possibly valid enums that can then be filtered out by version and other factors like extension is going to be the most useful form of these enum groups.

Reply:
@NogginBops are you okay with this PR? I'm not sure if you're requesting a change, or just pointing out gl.xml is not perfect yet.

Reply:
As I think the groups discussion will result in us looking at these groups more thoroughly again (and hopefully fix them), I think there is not much harm in merging this. So yes, I think this can be merged as is.